group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -3984,6 +3984,8 @@ CoordinatorResult<Void, CoordinatorRecord> classicGroupJoinToClassicGroup(
 
```

Comment:
We have another case in this file where we delete a classic group. Take a look at `maybeDeleteEmptyClassicGroup` method.